src/MusicStore/Program.cs

```diff
@@ -14,9 +14,11 @@ public static class Program
         public static void Main(string[] args)
         {
             var totalTime = Stopwatch.StartNew();
+            MusicStoreEventSource eventSource = new MusicStoreEventSource();
+            eventSource.ServerStartupBegin();
```

Comment:
Isn't the typical way to do this to use a static?

Reply:
Yes, it is. I just wrote something slightly simpler because I had no need of the static instance.